tests/string.in

```diff
@@ -116,6 +116,8 @@ string repeat -n10 -m4 'foo'
 
 string repeat -n10 --max 5 'foo'
 
+string repeat -n3 -m20 'foo'
```

Comment:
I feel like my other tests would catch such a bug, but i could add a length check on the output just to make sure and make it clear this is that exact class of bug we're testing